### ISSUE TITLE
mpicc can't build simple program on Travis Mac image

### ISSUE THREAD

**Issue by ChristopherHogan** [conda-forge/mpich-feedstock#29]
Issue:
Since `osx-64/mpich-3.2.1-h26a2512_6.tar.bz2` was uploaded to anaconda.org on 11/19/18, my conda recipes that use `mpicc` have been broken. Apologies if this is a Travis issue.

To reproduce, run the following on Travis with the default osx image (MacOS 10.13 and Xcode 9.4.1).
```bash
$ conda create -n mpich_6 -c conda-forge mpich
$ source activate mpich_6
$ echo "int main() { return 0; }" > main.c
$ mpicc main.c

clang: warning: no such sysroot directory: '/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk' [-Wmissing-sysroot]
ld: library not found for -lc++
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```
If I use the previous build, `osx64/mpich-3.2.1-h2612512_5`, everything works as expected:
```bash
$ conda create -n mpich_5 -c conda-forge mpich=3.2.1=h26a2512_5
$ source activate mpich_5
$ echo "int main() { return 0; }" > main.c
$ mpicc main.c
# success
```

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# packages in environment at /Users/travis/miniconda/envs/mpich_6:
#
# Name                    Version                   Build  Channel
libgfortran               3.0.0                         1    conda-forge
mpi                       1.0                       mpich    conda-forge
mpich                     3.2.1                h26a2512_6    conda-forge

```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info
   active environment : mpich_6
    active env location : /Users/travis/miniconda/envs/mpich_6
            shell level : 1
       user config file : /Users/travis/.condarc
 populated config files : /Users/travis/.condarc
          conda version : 4.5.11
    conda-build version : 3.16.3
         python version : 3.7.1.final.0
       base environment : /Users/travis/miniconda  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/osx-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/pro/osx-64
                          https://repo.anaconda.com/pkgs/pro/noarch
          package cache : /Users/travis/miniconda/pkgs
                          /Users/travis/.conda/pkgs
       envs directories : /Users/travis/miniconda/envs
                          /Users/travis/.conda/envs
               platform : osx-64
             user-agent : conda/4.5.11 requests/2.19.1 CPython/3.7.1 Darwin/17.4.0 OSX/10.13.3
                UID:GID : 501:20
             netrc file : None
           offline mode : False

```
</details>

**Comment by minrk** [comment]
I believe this will be fixed by build number 7 (#30) which should generally be available within an hour or so. Thanks for opening the Issue!

**Comment by ChristopherHogan** [comment]
Thanks for the suggestions and quick fix.  Unfortunately my recipe is in a private repo (not because there's anything secret about it, but so I can access ssh debugging on Travis Enterprise). But I'd be happy to post a gist if you want more info.
